src/main/scala/au/com/agiledigital/toolform/command/generate/kubernetes/VolumeClaimWriter.scala

```diff
@@ -0,0 +1,106 @@
+package au.com.agiledigital.toolform.command.generate.kubernetes
+
+import au.com.agiledigital.toolform.model.{Resource}
+import collection.JavaConversions._
+import collection.JavaConverters._
```

Comment:
I'm getting a few unused import and deprecation warnings in this file when I run `sbt test`